test/DollarTest.cpp

```diff
@@ -0,0 +1,15 @@
+#include "gtest/gtest.h"
+
+#include "Dollar.h"
+
+using namespace testing;
```

Comment:
I think there's no need to use this namespace here. Is there any that I'm not known of?